vegaprotocol/approbation
# Add globs for check-codes and check-filenames

Un-hardcodes paths for 2/3 scripts. Also adds tests for those scripts

- Add glob dep, test folders
- Partial migration - paths still hardcoded
- Quick function change to allow for a local run
- Make check-filenames accept blocks
- Fix filename check to properly follow the glob (specifically file extensions
- Globs on codes
- Minor fixes
- Add github test action

Closes #3
